Jenkinsfile

```
@@ -1,4 +1,6 @@
-buildPlugin(useContainerAgent: true, configurations: [
-  [platform: 'linux', jdk: 17],
-  [platform: 'windows', jdk: 11],
+buildPlugin(
+  useContainerAgent: true,
+  configurations: [
+    [platform: 'linux', jdk: 21],
```

```suggestion
    [platform: 'linux', jdk: 11],
    [platform: 'linux', jdk: 21],
```

as per https://github.com/jenkinsci/archetypes/issues/650#issuecomment-1729897760